test/test_gdal_tutorials.jl

```diff
@@ -1,130 +1,227 @@
-                132.0f0,132.0f0,140.0f0,132.0f0,132.0f0,132.0f0,107.0f0,
-                132.0f0,107.0f0,132.0f0,132.0f0,107.0f0,123.0f0,115.0f0,
-                156.0f0,148.0f0,107.0f0,132.0f0,107.0f0,115.0f0,99.0f0,
-                123.0f0,99.0f0,74.0f0,115.0f0,82.0f0,115.0f0,115.0f0,
-                107.0f0,123.0f0,123.0f0,99.0f0,123.0f0,123.0f0,115.0f0,
-                115.0f0,107.0f0,90.0f0,99.0f0,107.0f0,107.0f0,99.0f0,
-                123.0f0,107.0f0,140.0f0,123.0f0,123.0f0,115.0f0,99.0f0,
-                132.0f0,123.0f0,115.0f0,115.0f0,123.0f0,132.0f0,115.0f0,
-                123.0f0,132.0f0,214.0f0,156.0f0,165.0f0,148.0f0,115.0f0,
-                148.0f0,156.0f0,148.0f0,140.0f0,165.0f0,156.0f0,197.0f0,
-                156.0f0,197.0f0,140.0f0,173.0f0,156.0f0,165.0f0,148.0f0,
-                156.0f0,206.0f0,214.0f0,181.0f0,206.0f0,173.0f0,222.0f0,
-                206.0f0,255.0f0,214.0f0,173.0f0,214.0f0,255.0f0,214.0f0,
-                247.0f0,255.0f0,230.0f0,206.0f0,197.0f0]
-
-        @test AG.metadatadomainlist(dataset) == ["IMAGE_STRUCTURE", "", "DERIVED_SUBDATASETS"]
-        @test AG.metadata(dataset) == ["AREA_OR_POINT=Area"]
-        @test AG.metadataitem(dataset, "AREA_OR_POINT") == "Area"
-        @test AG.metadata(dataset, domain = "IMAGE_STRUCTURE") == ["INTERLEAVE=BAND"]
-        @test AG.metadata(dataset, domain = "") == ["AREA_OR_POINT=Area"]
-        @test AG.metadata(dataset, domain = "DERIVED_SUBDATASETS") == [
-            "DERIVED_SUBDATASET_1_NAME=DERIVED_SUBDATASET:LOGAMPLITUDE:data/utmsmall.tif",
-            "DERIVED_SUBDATASET_1_DESC=log10 of amplitude of input bands from data/utmsmall.tif"
-        ]
-    end
+import ArchGDAL;
+const AG = ArchGDAL;
+
+@testset "test_gdal_tutorials.jl" begin
+    @testset "Raster Tutorial" begin
+        driver = AG.getdriver("GTiff")
+
+        AG.read("data/utmsmall.tif") do dataset
+            @test AG.shortname(driver) == "GTiff"
+            @test AG.longname(driver) == "GeoTIFF"
+            @test AG.width(dataset) == 100
+            @test AG.height(dataset) == 100
+            @test AG.nraster(dataset) == 1
+
+            nad27_prefix = "PROJCS[\"NAD27 / UTM zone 11N\",GEOGCS[\"NAD27\",DATUM[\"North_American_Datum_1927\","
+            @test startswith(AG.getproj(dataset), nad27_prefix) == true
+            @test AG.getgeotransform(dataset) ≈
+                  [440720.0, 60.0, 0.0, 3.75132e6, 0.0, -60.0]
+
+            band = AG.getband(dataset, 1)
+            @test AG.blocksize(band) ≈ [100, 81]
+            @test AG.pixeltype(band) == UInt8
+            @test AG.getname(AG.getcolorinterp(band)) == "Gray"
+
+            @test AG.minimum(band) ≈ 0.0
+            @test AG.maximum(band) ≈ 255.0
+
+            @test AG.noverview(band) == 0
+
+            # Reading Raster Data
+            @test AG.width(band) == 100
+            data = map(Float32, AG.read(dataset, 1))
+            @test data[:, 1] ≈ Float32[
+                107.0f0,
+                123.0f0,
```

Comment:
Also https://domluna.github.io/JuliaFormatter.jl/dev/skipping_formatting/ maybe?

Reply:
Reflected in https://github.com/yeesian/ArchGDAL.jl/pull/179/commits/e86beac5117657411ed42ff1f269003db0bd6f69, thanks!